### PR TITLE
Fix issue 29937 - Array to string conversion in information of cli installation command

### LINKED ISSUE #29937: Displaying informations from installation command from cli : Array to String conversion PHP notice
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Displaying informations from installation command on PrestaShop 8.x from the command line throws a PHP Notice: 

```
php index_cli.php
Notice: Array to string conversion in /path/install-dev/classes/controllerConsole.php on line 95
```

### Expected behavior

Installing PrestaShop from the command line should succeed without any errors, warning, or notices.

### Steps to reproduce

1. Download the zip and extract PrestaShop.

2. Install composer dependencies (if needed)

3. Go to install-dev (or install)

4. Run php index_cli.php

Note that in PrestaShop 1.7.8.x the following command succeeds without any notices.

### PrestaShop version(s) where the bug happened

8.x

### PHP version(s) where the bug happened

7.4

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by thomasnares**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fixes display of default array parameter in cli
| Type?             | bug fix
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #29937 
| How to test?      | Go to install-dev directory, run php index_cli.php

### CODE REVIEW

File: install-dev/classes/controllerConsole.php [hunk 1]
```diff
@@ -91,7 +91,12 @@ final public static function execute($argc, $argv)
             echo 'Arguments available:' . PHP_EOL;
             foreach ($available_arguments as $key => $arg) {
                 $name = isset($arg['name']) ? $arg['name'] : $key;
-                echo '--' . $name . "\t" . (isset($arg['help']) ? $arg['help'] : '') . (isset($arg['default']) ? "\t" . '(Default: ' . $arg['default'] . ')' : '') . PHP_EOL;
+                echo '--'
+                    . $name
+                    . "\t"
+                    . (isset($arg['help']) ? $arg['help'] : '')
+                    . (isset($arg['default']) ? "\t" . '(Default: ' . (is_array($arg['default']) ? '[' . implode(',', $arg['default']) . ']' : $arg['default']) . ')' : '')
+                    . PHP_EOL;
```
Comment (mflasquin) [inline install-dev/classes/controllerConsole.php]: Good but I think It's a little difficult to read (two ternary operator in one line), I purpose to transform the code like this :

```suggestion
$help = isset($arg['help']) ? $arg['help'] : '';
$default = '';

if(isset($arg['default'])) {
    if(is_array($arg['default'])) {
        $default = '(Default: [' . implode(',', $arg['default']) . '])';
    } else {
        $default = '(Default: ' . $arg['default'] . ')';
    }
}

echo '--' . $name . "\t" . $help . "\t" . $default . PHP_EOL;
```

What do you think about it ? :)

**Review by mflasquin — CHANGES_REQUESTED** [review]

**Comment by thomasnares** [comment]
thanks @mflasquin, looks better :)

**Comment by mflasquin** [comment]
> thanks @mflasquin, looks better :)

You're welcome, but take care to the indentation I think my updates broked it ;)

**Comment by thomasnares** [comment]
Fixed 👍

**Review by atomiix — APPROVED** [review]

**Review by mflasquin — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
@thomasnares 💪 🎉

**Comment by paulnoelcholot** [comment]
Hello @thomasnares , 

Thanks for your PR. The issue is welle fixed and I found no regression :tada: 

 
image

**Review by paulnoelcholot — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by jolelievre** [comment]
Thanks @thomasnares and @paulnoelcholot

**Pull request merged**